writers/streamingbatchwriter/streamingbatchwriter.go

```diff
@@ -182,26 +182,28 @@ func (w *StreamingBatchWriter) Write(ctx context.Context, msgs <-chan message.Wr
 	errCh := make(chan error)
 	defer close(errCh)
 
-	go func() {
-		for err := range errCh {
-			w.logger.Err(err).Msg("error from StreamingBatchWriter")
-		}
-	}()
+	for {
+		select {
+		case msg := <-msgs:
+			if msg == nil {
```

Comment:
Wouldn't you want to `msg, ok :=` instead here?

Reply:
Yeah that would be clearer, fixed in https://github.com/cloudquery/plugin-sdk/pull/1913/commits/512d6c0af379dd8b31b50a47c2071933f6946731